navigator/backend/src/test/scala/com/digitalasset/navigator/backend/query/ContractSortSpec.scala

```diff
@@ -81,13 +89,18 @@ class ContractSortSpec extends FlatSpec with Matchers {
     }
   }
 
+  implicit val sortParties: Ordering[Seq[String]] =
+    Ordering.fromLessThan[Seq[String]](_.mkString < _.mkString)
+
   test(List(), contracts)
   test(List("id" -> ASCENDING), contracts.sortBy(_.id.unwrap))
   test(List("id" -> DESCENDING), contracts.sortBy(_.id.unwrap)(Ordering[String].reverse))
   test(
     List("agreementText" -> ASCENDING),
     contracts.sortBy(_.agreementText)
   )
+  test(List("signatories" -> ASCENDING), contracts.sortBy(_.signatories))
+  test(List("observers" -> ASCENDING), contracts.sortBy(_.signatories))
```

Comment:
```suggestion
  test(List("observers" -> ASCENDING), contracts.sortBy(_.observers))
```

Reply:
Wow, this is very bad, thanks for catching this.